libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -722,7 +719,6 @@ async def _astream(
             project=self.project,
             convert_system_message_to_human=self.convert_system_message_to_human,
         )
-        message = history_gemini.pop()
```

Comment:
The variable message is currently unused.

It should not be removed from the history as generate_content_async requires the user message to be present in the contents parameter passed to the client, see https://ai.google.dev/api/python/google/generativeai/GenerativeModel#multi-turn.